dev/DynamicDependency/PackageGraphManager.cpp

```diff
@@ -65,6 +65,12 @@ HRESULT MddCore::PackageGraphManager::GetCurrentPackageInfo2(
     UINT32* count,
     GetCurrentPackageInfo2Function getCurrentPackageInfo2) noexcept try
 {
+    // Check parameter(s) per GetCurrentPackageInfo2 compatibility
+    //   * bufferLength is required
+    //   * buffer is required if bufferLength >0
+    //   * count is required if buffer is not nullptr
+    RETURN_HR_IF(E_INVALIDARG, (bufferLength == nullptr) || (*bufferLength > 0 && buffer == nullptr) || ((count == nullptr) && (buffer != nullptr)));
```

Comment:
Nit: Horizontal scrollbar when diffing :(  Maybe wrap it?

Reply:
The maxlen=100 rule is for *.md (specs), not code.